src/assets/icons/generate_icon_sizes.sh

```diff
@@ -0,0 +1,13 @@
+convert -geometry x76 icon-512x512.png apple-icon-76x76.png
+convert -geometry x120 icon-512x512.png apple-icon-120x120.png
+convert -geometry x152 icon-512x512.png apple-icon-152x152.png
+convert -geometry x167 icon-512x512.png apple-icon-167x167.png
+convert -geometry x180 icon-512x512.png apple-icon-180x180.png
+
+convert -geometry x96 icon-512x512.png icon-96x96.png
+convert -geometry x72 icon-512x512.png icon-72x72.png
+convert -geometry x128 icon-512x512.png icon-128x128.png
+convert -geometry x144 icon-512x512.png icon-144x144.png
+convert -geometry x152 icon-512x512.png icon-152x152.png
+convert -geometry x192 icon-512x512.png icon-192x192.png
+convert -geometry x384 icon-512x512.png icon-384x384.png
```

Comment:
What kind of magic is that? Does that script generate all the icon sizes based on a single image?
Does that .sh file work on Windows?

Reply:
Funny you should ask about magic. The 'convert' command is from a library called ImageMagick: https://www.imagemagick.org. There's a Windows version, but the .sh script is for running on Linux.

Reply:
well, I'm not sure I like the fact that we introduce tools that are going to run only on a particular platform.
There are plenty NodeJs tools out there for image processing. For example https://github.com/lovell/sharp
Why don't we use that instead?

Reply:
.sh will run on Windows with this: http://win-bash.sourceforge.net. Why don't we merge this PR and if we need to update the icons again, then we'll pull in the node tool.